packages/components/src/DataViewer/Text/SimpleTextKeyValue/SimpleTextKeyValue.test.js

```diff
@@ -15,6 +15,20 @@ describe('SimpleTextKeyValue', () => {
 		);
 		expect(wrapper.getElement()).toMatchSnapshot();
 	});
+
+	it('should render the key by the talend.component.label property', () => {
+		const wrapper = shallow(
+			<SimpleTextKeyValue
+				className="myCLass"
+				formattedKey="non readable value"
+				schema={{ 'talend.component.label': 'readable value' }}
+				value="myValue"
+				separator=" : "
+				style={{ padding: 0 }}
+			/>,
+		);
+		expect(wrapper.getElement()).toMatchSnapshot();
```

Comment:
Please don't snapshot a whole component to check that a simple part is the string we want.
This introduces a lot of noise in the test and debug is more difficult.

Select the part you're interested in, then `expect()` `part.text()` to be the text you expect.